spring-integration-jmx/src/test/java/org/springframework/integration/jmx/config/MBeanExporterParserTests.java

```diff
@@ -133,8 +129,10 @@ private void checkCustomized(org.springframework.integration.support.management.
 	public static class CustomMetrics implements org.springframework.integration.support.management.MetricsFactory {
 
 		@Override
-		public AbstractMessageChannelMetrics createChannelMetrics(String name) {
-			return new DefaultMessageChannelMetrics(name,
+		public org.springframework.integration.support.management.AbstractMessageChannelMetrics
```

Comment:
Same here: can we just remove all the tests code with those deprecated classes and methods?